src/containers/Tablet/TabletTable/TabletTable.tsx

```diff
@@ -33,6 +33,33 @@ const columns: Column<ITabletPreparedHistoryItem>[] = [
             return row.leader ? 'leader' : row.followerId;
         },
     },
+    {
+        name: 'Node ID',
+        align: DataTable.RIGHT,
+        sortable: false,
+        render: ({row}) => {
+            return <InternalLink to={getDefaultNodePath(row.nodeId)}>{row.nodeId}</InternalLink>;
+        },
+    },
+    {
+        name: 'Node FQDN',
+        sortable: false,
+        render: ({row}) => {
+            if (!row.fqdn) {
+                return <span>—</span>;
+            }
+            return (
+                <div className={b('host')}>
```

Comment:
I don't know the reason, but there is some issue with width defined in columns. It works only when you set table width. Here I don't want to set table width. The second reason - in case of set column width, it has not desired overflow behaviour (it's just hidden, overflow by EntityStatus component doesn't apply), so for long FQDN copy button is hidden.